.github/workflows/bigquery.yml

```diff
@@ -90,10 +91,10 @@ jobs:
       - name: Run deploy
         run: |
           cd clouds/bigquery
-          make deploy diff="$GIT_DIFF" production=1
+          make deploy diff="$GIT_DIFF"
```

Comment:
Removing production=1 from here might be the reason why you were getting a versión including `-dev`.

Reply:
No, because I was getting it in the test, that is before this step. 
The reason was that it was doing a substring of the first 10 characters, to get the version in format yyyy-MM-dd, but in the new format x.y.z that was not valid. I think that's already fixed in the tests